### PR TITLE
Release 1.0.0-alpha.1

### PR DESCRIPTION
**Pull request by enyachoke**

### CODE REVIEW

**Review by mks-d — COMMENTED** [review]
@enyachoke don't you need to do a double commit, the second one reverting to bleeding edge?

**Comment by enyachoke** [comment]
> @enyachoke don't you need to do a double commit, the second one reverting to bleeding edge?

Yes, But I let it like this to begin with so the changes are obvious.

File: .env [hunk 1]
```diff
@@ -120,3 +120,8 @@ OPENMRS_CONFIG_CHECKSUMS_PATH=
 MYSQL_DATADIR=
 POSTGRES_DATADIR=
 ODOO_FILESTORE=
+
+#
+# Frontend tag
+#
+TAG=3.0.0-beta.2
```
Comment (mks-d) [inline .env:127]: Why's that, where's this used? Looks like this is not a good naming anyway, it's too vague. If we need to keep it it should become something like `O3_FRONTEND_TAG` no?

Comment (enyachoke) [reply .env]: I have removed it

File: docker-compose.yml [hunk 2]
```diff
@@ -62,7 +62,7 @@ services:
 
   # OpenMRS
   openmrs:
-    image: mekomsolutions/openmrs:latest
+    image: mekomsolutions/openmrs:69f1dce
```
Comment (mks-d) [inline docker-compose.yml:65]: Taking this one as an example, is there no way to not use hashes but rather a user-friendly version (like it's done for [`openmrs/openmrs-reference-application-3-frontend:3.0.0-beta.2`](https://github.com/ozone-his/ozone-docker/pull/28/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R189))?

File: .env (hunk 1 above)
Comment (Ruhanga) [reply .env]: I've demonstrated it here https://github.com/ozone-his/ozone-docker/pull/30. Could be merged if okay.

**Review by mks-d — APPROVED** [review]

**Pull request merged**